.github/CODEOWNERS

```diff
@@ -70,11 +70,14 @@
 /addons/mrp_subcontracting_account/ @odoo/rd-logistics
 /addons/mrp_subcontracting_dropshipping/ @odoo/rd-logistics
 
+/addons/payment*/ @odoo/rd-payment
```

Comment:
This will shadow the @odoo/rd-website rule on ir_http wildcard in this module, need to see with them if it's ok, or if both ownership have to be kept by adding a more specific rule for it.

Reply:
If I understand correctly, since I moved the payment rule before the website one, changes to `payment/models/ir_http.py` will only ping the website team ?
@AntoineVDV do you want to also maintain the ir_http file ? :D

Reply:
Yes you got that correctly, depending on the placement which ever rule is matching last will take priority.

Reply:
It's fine as it is now :)